SEC/AllSEC-ExampleIndividuals.rdf

```diff
@@ -69,10 +69,10 @@
 		<sm:moduleAbbreviation>fibo-sec</sm:moduleAbbreviation>
 		<rdfs:seeAlso rdf:resource="https://www.edmcouncil.org/fibo/"/>
 		<owl:imports rdf:resource="http://www.omg.org/techprocess/ab/SpecificationMetadata/"/>
-		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/AllFBC-NorthAmericanExamples/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/AllFBC-ExampleIndividuals/"/>
```

Comment:
No idea what this change has to do with the PR

Reply:
Added to assist with the testing that I did, which was extensive.  Seemed like it was useful so I saved it.  The challenge is that AllProd is too big to do incremental testing with, and you saw the results of the reasoning testing, attached to this resolution, above.